src/common/__test__/string-builder.spec.js

```diff
@@ -120,3 +120,17 @@ test('combineURLs should return the relative URL when the base URL is undefined'
 
     expect(result).toBe('https://github.com/users/hg-pyun');
 });
+
+test('combineURLs should return the relative URL when the base URL have a path', () => {
+    const sb = new StringBuilder(getGlobalConfig());
+    const result = sb.combineURLs('https://github.com/users', '/hg-pyun');
+
+    expect(result).toBe('https://github.com/users/hg-pyun');
+});
+
+test('combineURLs should return the relative URL when the base URL have a complex path', () => {
+    const sb = new StringBuilder(getGlobalConfig());
+    const result = sb.combineURLs('https://github.com/foo/bar/', '//hg-pyun');
```

Comment:
`//` is right?

```suggestion
    const result = sb.combineURLs('https://github.com/foo/bar/', '/hg-pyun');
```

Reply:
yes, path.join must build correct path
in all cases: a) baseUrl ends with / and relativeUrl starts with /, b) baseUrl ends with / and relativeUrl starts with //, c) a) baseUrl ends with / and relativeUrl starts without /